code/src/components/GameBoardHeader.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { useSelector } from 'react-redux';
+
+export const GameBoardHeader = () => {
+  const currentGameState = useSelector(store => store.gameReducer.gameDetails);
+  const locationDescription = currentGameState.description;
+  const playerName = useSelector(store => store.gameReducer.userName);
+
+  return(
+    <div className="gameboard-current-details">
```

Comment:
This could be a header instead of a div. Also consider switching to an h1 in the header and import this to the start-game-board with a conditional rendering instead? Like, if game hasn't started the h1 should say Maze adventure, and if it has started you could display the directions in it 😊